docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,132 @@
+<%(TOC)%>
+# Customising player creation
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+  * [Basic player spawning]({{urlRoot}}/modules/player-lifecycle/basic-player-creation)
+  * [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+## Manually sending player creation requests
+
+By default, the Player Lifecycle Feature Module sends a player creation request is sent as soon as the client-worker instance connects to SpatialOS.
+
+To modify this behaviour, you must set `PlayerLifecycleConfig.AutoRequestPlayerCreation` to false by one of the following ways:
+
+* Add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, autoRequestPlayerCreation: false);`
+* Set the field to false manually during set-up: `PlayerLifecycleConfig.AutoRequestPlayerCreation = false;`
+
+Once `AutoRequestPlayerCreation` is set to false you must manually call the `RequestPlayerCreation` method, which can be found in the `SendCreatePlayerRequestSystem`.
+
+```csharp
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+playerCreationSystem.RequestPlayerCreation();
+```
+
+To find out more about the `RequestPlayerCreation` method, take a look at the `SendCreatePlayerRequestSystem` [API reference documentation]({{urlRoot}}/api/player-lifecycle/send-create-player-request-system#methods).
+
+## Adding a player creation callback
+
+If manual player creation is enabled, you can configure a callback to be triggered when a player creation response is received. The callback must be of type `Action<PlayerCreator.CreatePlayer.ReceivedResponse>`.
+
+For example:
+
+```csharp
+public void CallPlayerCreation()
+{
+    var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+    playerCreationSystem.RequestPlayerCreation(callback: OnCreatePlayerResponse);
+}
+
+private void OnCreatePlayerResponse(PlayerCreator.CreatePlayer.ReceivedResponse response)
+{
+    if (response.StatusCode != StatusCode.Success)
+    {
+        Debug.LogWarning($"Error: {response.Message}");
+    }
+}
+```
+
+## Using arbitrary data in the player creation loop
```

Comment:
Similarly, should this be a sub header? Not entirely sure

Reply:
I don't think it should because there is more to this than calling `RequestPlayerCreation`. Even if there is some overlap, it probably makes sense to keep the information about the `playerCreationArguments`/how to serialize/deserialize under a separate header